packages/twenty-server/src/modules/messaging/message-import-manager/services/messaging-import-exception-handler.service.ts

```diff
@@ -230,8 +229,15 @@ export class MessageImportExceptionHandlerService {
     messageChannel: Pick<MessageChannelWorkspaceEntity, 'id'>,
     workspaceId: string,
   ): Promise<void> {
-    if (syncStep === MessageImportSyncStep.FULL_MESSAGE_LIST_FETCH) {
-      return;
+    if (syncStep === MessageImportSyncStep.MESSAGE_LIST_FETCH) {
+      await this.handleUnknownException(
```

Comment:
this should never happen (I think we should actually refactor this part to stop trying to parse message-import and message-list-fetch exception in one place, they are two differents). Right now the logic is in common and NotFound should never happen in case of message list fetch